migration/sql-files/034-work-items-history.sql

```diff
@@ -0,0 +1,36 @@
+-- create a revision table for work items, using the some columns + identity of the user and timestamp of the operation
+CREATE TABLE work_item_revisions (
+    id bigint NOT NULL,
+    revision_time timestamp with time zone default current_timestamp,
+    revision_type int NOT NULL,
+    modifier_id uuid NOT NULL,
+    work_item_id bigint NOT NULL,
+    work_item_type text,
+    work_item_version integer,
+    work_item_fields jsonb
+);
+
+CREATE SEQUENCE work_item_revisions_id_seq
```

Comment:
I'm guessing it's using sequence here to preserve some kind of order of revisions. revision_time could almost safely be used for the same .. :)